src/components/views/elements/Dropdown.js

```diff
@@ -115,7 +115,7 @@ export default class Dropdown extends React.Component {
 
     componentWillReceiveProps(nextProps) {
         this._reindexChildren(nextProps.children);
-        const firstChild = React.Children.toArray(nextProps.children)[0];
+        const firstChild = nextProps.children[0];
```

Comment:
TBH, if it is still an array with one child that's probably fine - I'm not too fussed about supporting dropdowns with no options.